src/extension.ts

```diff
@@ -19,9 +19,7 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
     ext.ignoreBundle = ignoreBundle;
     ext.outputChannel = createAzExtOutputChannel('Azure Static Web Apps', ext.prefix);
     context.subscriptions.push(ext.outputChannel);
-    ext.ui = new AzureUserInput(context.globalState);
```

Comment:
If you're going to remove the extension-scoped `ui` in this PR, you also need to remove it  from `extensionVariables.ts` and you need to bump the ui/appservice npm packages

I did it in a separate PR in the case of storage (https://github.com/microsoft/vscode-azurestorage/pull/988), but up to you if you want to combine it here